src/main/java/data/FileParser.java

```diff
@@ -0,0 +1,42 @@
+package data;
+
+import java.io.File;
+import java.util.*;
+
+public class FileParser {
+  private static final String DEFAULT_FILE_PATH = "../../src/main/resources";
```

Comment:
I currently don't have that filepath, so I'm assuming you created this to satisfy the testing problems you were having.

Reply:
yep. maven also sort of prescribes this filestructure, so that is where I've been putting my sample files for testing